src/main/com/example008/radek/ExtendedCalculatorApplication.java

```diff
@@ -0,0 +1,15 @@
+package main.com.example008.radek;
+
+public class ExtendedCalculatorApplication {
+
+    public static void main(String[] args) {
+
+        ExtendedCalculatorV1 calculator = new ExtendedCalculatorV1();
+        System.out.println(calculator.isThisNumberOdd(2.2));
+        System.out.println(calculator.plus(5, 2));
+        System.out.println(calculator.square(2));
+        System.out.println(calculator.squareRoot(25));
+
```

Comment:
proc ty prazdne tradky na konci? v dalsi classe opet nesjednocene to odradkovani 
bylo by fajn to pred vytvorenim PR jeste projit jednou

Reply:
Přeformátováno